iceoryx_posh/include/iceoryx_posh/popo/port_queue_policies.hpp

```diff
@@ -23,36 +23,30 @@ namespace iox
 {
 namespace popo
 {
-/// @brief Used by publisher
-/// @todo iox-#27 replace with ConsumerTooSlowPolicy
-enum class SubscriberTooSlowPolicy : uint8_t
-{
-    WAIT_FOR_SUBSCRIBER,
-    DISCARD_OLDEST_DATA
-};
-/// @brief Used by subscriber
-/// @todo iox-#27 replace with QueueFullPolicy2
-enum class QueueFullPolicy : uint8_t
-{
-    BLOCK_PUBLISHER,
-    DISCARD_OLDEST_DATA
-};
-
 /// @brief Used by producers how to adjust to slow consumer
 enum class ConsumerTooSlowPolicy : uint8_t
 {
     /// Waits for the consumer it it's queue is full
     WAIT_FOR_CONSUMER,
+    /// @deprecated Compatibility value for the legacy use with publisher only. Will be remove in a future release.
+    /// Please use `WAIT_FOR_CONSUMER` instead.
+    WAIT_FOR_SUBSCRIBER = WAIT_FOR_CONSUMER,
     /// Discards the oldest data and pushes the newest one into the queue
     DISCARD_OLDEST_DATA
 };
+/// @deprecated Compatibility alias for the legacy use with publisher only. Will be remove in a future release.
+/// Please use `ConsumerTooSlowPolicy` instead.
+/// @todo iox-#27 decide whether this should be hard deprecated with [[deprecated]] attribute for the next release
```

Comment:
From my point of view this can be marked as `[[deprecated]]` since we are striving for the next major release where API breaking changes are allowed

Reply:
I was not sure about this, so I thought it would be best to let the reviewer decide.

This means I should also mark the `SubscriberTooSlowPolicy` C++ enum as `[[deprecated]]`?